front-packages/akeneo-design-system/src/components/Tiles/Tiles.tsx

```diff
@@ -6,30 +6,47 @@ import {Key, Override} from '../../';
 
 type Size = 'small' | 'big';
```

Comment:
```suggestion
type Size = 'small' | 'big' | 'inline';
```
I believe Stéphane said that the inline was supposed to be on the same existing `size` prop, as we cannot have a big & inline tile so size="big" with inline="true" would make no sense